doc/userguide/rules/http-keywords.rst

```diff
@@ -60,8 +60,8 @@ http_header                    Modifier                 Both
 http_raw_header                Modifier                 Both
 http_cookie                    Modifier                 Both
 http_server_body               Modifier                 Response
-http.server                    Modifier                 Response
-http.location                  Modifier                 Response
+http.server                    Sticky Buffer            Response
```

Comment:
this is not related to the analyzer, so should go into its own commit

Reply:
Will do.